cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -624,10 +625,51 @@ update msg model =
                     )
 
                 ShowSaveTestError ->
-                    ({model | showSaveTestError = True}, Cmd.none)
+                    ( { model | showSaveTestError = True }, Cmd.none )
 
         ServerGetAttributeResponse attributes ->
-            ( { model | customAttributes = attributes }
+            let
+                -- We also want set the attributeStates once the attributes are fetched
+                attributeEditorState : SDKDict.Dict AttrValueDetail ValueEditor.EditorState
+                attributeEditorState =
+                    attributes
+                        |> Dict.toList
+                        |> List.map
+                            (\( customAttributeId, customAttributeDetail ) ->
+                                let
+                                    selectedModuleName =
+                                        model.homeState.selectedModule
+                                            |> Maybe.map Tuple.second
+                                            |> Maybe.withDefault []
```

Comment:
Empty module name as the default? That looks error-prone.

Reply:
This have been fixed. So we only update the attributeStates only when a module is selected.